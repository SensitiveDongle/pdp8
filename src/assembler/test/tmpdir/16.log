Undefined symbol some_var used at line 3
Undefined symbol subroutine used at line 2
*** 2 errors in assembly
