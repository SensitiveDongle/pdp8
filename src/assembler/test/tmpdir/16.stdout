  0x000: 0x000    1: program, 0
  0x001: 0xA00    2: 		JMP subroutine
  0x002: 0x200    3: 		TAD some_var
  0x003: 0xF02    4: 		HLT
                  5: 
                  6: END program
